moveit_experimental/jog_arm/src/jog_arm/jog_calcs.cpp

```diff
@@ -635,7 +631,9 @@ void JogCalcs::halt(trajectory_msgs::JointTrajectory& jt_traj)
   {
     // For position-controlled robots, can reset the joints to a known, good state
     if (parameters_.publish_joint_positions)
+      {
```

Comment:
Indentation looks wrong here and the next couple lines. See the moveit contributors guide about clang-format -- it will fix this for you.

https://moveit.ros.org//documentation/contributing/code/